### PR TITLE
Changed the way to search for disks for 'OSX', 'Linux' and 'android' …

### PR DESCRIPTION
**Pull request by Neizvestnyj**
I decided to do a disk search without using the subprocess module, the mount command is more universal as for me, the new method will work even if there are more than 2 spaces in the disk name.

### CODE REVIEW

File: kivymd/uix/filemanager.py [hunk 1]
```diff
@@ -653,8 +658,8 @@ def get_content(self):
                                 pass
                         else:
                             if (
-                                not self.show_hidden_files
-                                and content.startswith(".")
+                                    not self.show_hidden_files
+                                    and content.startswith(".")
```
Comment (HeaTTheatR) [inline kivymd/uix/filemanager.py]: This is not according to the `PEP8` standard.

File: kivymd/uix/filemanager.py [hunk 2]
```diff
@@ -636,7 +641,7 @@ def get_content(self):
                 if os.path.isdir(os.path.join(self.current_path, content)):
                     if self.search == "all" or self.search == "dirs":
                         if (not self.show_hidden_files) and (
-                            content.startswith(".")
+                                content.startswith(".")
```
Comment (HeaTTheatR) [inline kivymd/uix/filemanager.py]: This is not according to the `PEP8` standard.

File: kivymd/uix/filemanager.py [hunk 3]
```diff
@@ -557,8 +562,8 @@ def show(self, path):
                 )
             for name_file in self.__sort_files(files):
                 if (
-                    os.path.splitext(os.path.join(path, name_file))[1]
-                    in self.ext
+                        os.path.splitext(os.path.join(path, name_file))[1]
+                        in self.ext
```
Comment (HeaTTheatR) [inline kivymd/uix/filemanager.py]: This is not according to the `PEP8` standard.

File: kivymd/uix/filemanager.py [hunk 4]
```diff
@@ -425,9 +423,9 @@ def __init__(self, **kwargs):
         toolbar_label = self.ids.toolbar.children[1].children[0]
         toolbar_label.font_style = "Subtitle1"
         if (
-            self.selector == "any"
-            or self.selector == "multi"
-            or self.selector == "folder"
+                self.selector == "any"
+                or self.selector == "multi"
+                or self.selector == "folder"
```
Comment (HeaTTheatR) [inline kivymd/uix/filemanager.py]: This is not according to the `PEP8` standard.

File: kivymd/uix/filemanager.py (hunk 1 above)
Comment (Neizvestnyj) [reply kivymd/uix/filemanager.py]: Hmm, PyCharm did it, not me)

File: kivymd/uix/filemanager.py (hunk 4 above)
Comment (Neizvestnyj) [reply kivymd/uix/filemanager.py]: I use autoformatting in my compiler, I will change it now

Comment (HeaTTheatR) [reply kivymd/uix/filemanager.py]: https://github.com/kivymd/KivyMD#setup-environment

**Comment by Neizvestnyj** [comment]
All fixed

**Pull request merged**